plugins/connectrpc/python/v0.4.2/requirements.txt

```diff
@@ -0,0 +1,14 @@
+aiohappyeyeballs==2.6.1
+aiohttp==3.12.15
+aiosignal==1.4.0
+attrs==25.3.0
+connect-python==0.4.2
+frozenlist==1.7.0
+idna==3.10
+multidict==6.6.3
+propcache==0.3.2
+protobuf==6.31.1
+protogen==0.3.1
+typing_extensions==4.14.1
+urllib3==2.5.0
+yarl==1.20.1
```

Comment:
for now, this includes all the deps of the entire connect-python, since the "compiler" (read: generator) is just an extra of the package.

I think we could probably trim this down, because the [plugin][1] imports the [generator][2], but otherwise seems self contained.

Generated this following the steps here: https://github.com/bufbuild/plugins/pull/688#discussion_r1279637498

[1]: https://github.com/connectrpc/connect-python/blob/v0.4.2/src/connectrpc/protoc_gen_connect_python/main.py
[2]: https://github.com/connectrpc/connect-python/blob/v0.4.2/src/connectrpc/generator.py